asterius/rts/rts.memory.mjs

```diff
@@ -56,6 +56,19 @@ export class Memory {
   i64LoadU16(p) { return BigInt(this.dataView.getUint16(Memory.unTag(p), true)); }
   heapAlloced(p) { return Memory.unTag(p) >= (this.staticMBlocks << Math.log2(rtsConstants.mblock_size)); }
   strlen(_str) { return this.i8View.subarray(Memory.unTag(_str)).indexOf(0); }
+  strLoad(_str) {
+      let p = Memory.unTag(_str);
+      let s = "";
+      let i = 0;
+      // NOTE: passing a non-null terminated string will give you garbage,
+      // as you deserve.
+      while(1) {
```

Comment:
Can we use a `for` loop here? We usually use `while(true)` only when the loop range is not trivial to calculate (e.g. walking a stack and stopping for specific frame types)

Reply:
Done.